public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -153,10 +152,11 @@ export class GeomapPanel extends Component<Props> {
     if (!this.map) {
       return;
     }
-    if (!cfg) {
-      cfg = { type: defaultGrafanaThemedMap.id };
+
+    if (!cfg?.type || config.geomapDisableCustomBaseLayer) {
```

Comment:
@eunice98k -- this will force using the server settings regardless of what is saved in the panel json